int64 Req_Flag
---
int64 Res_Flag
